### PR TITLE
Added a readme for v3 API

### PR DESCRIPTION
**Pull request by jkutner**
Basic instructions on how to run the v3 API.

### CODE REVIEW

File: v3/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```
+$ docker run --rm -v "$(pwd):/launch/app" -v "packs:/workspace" packs/v3:build
+```
+
+## Building the images
+
+To build the packs/v3 images yourself, you'll need Stephen's™️ YAML to JSON converter:
+
+```
+$ go get sclevine/yj
```
Comment (sclevine) [inline v3/README.md]: Should be `github.com/sclevine/yj`

File: v3/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Detect:
+
+```
+$ docker run --rm -v "$(pwd):/launch/app" -v "packs:/workspace" packs/v3:detect
+```
+
+Build:
+
+```
+$ docker run --rm -v "$(pwd):/launch/app" -v "packs:/workspace" packs/v3:build
```
Comment (sclevine) [inline v3/README.md]: I think this will modify the app directory in place.

File: v3/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Buildpack v3 reference implementation
+
+Execution of these images differs from the standard packs API.
+
+Optionally, you may want to create a Docker volume to share files across runs:
```
Comment (sclevine) [inline v3/README.md]: Probably want to add instructions for the export/analyze images as well. The `pack` CLI will be able to coordinate all of this very soon though.

File: v3/README.md (hunk 1 above)
Comment (jkutner) [reply v3/README.md]: fixed

File: v3/README.md (hunk 2 above)
Comment (jkutner) [reply v3/README.md]: oh, that's typically what i do. but I can change this to `$(pwd)/app` like in the root readme.

File: v3/README.md (hunk 3 above)
Comment (jkutner) [reply v3/README.md]: will do. how far away is "very soon"? these commands are complicated enough that i will forget them :)

**Comment by jkutner** [comment]
@sclevine this is ready

**Comment by sclevine** [comment]
Should we document analyze? It's necessary to make re-builds super quick.

**Comment by jkutner** [comment]
@sclevine I haven't figured out how to run analyze. I'm getting:

```
2018/08/29 15:32:58 Error: failed to parse arguments
```

I'll look into it, but let me know if you know what's wrong off the top of your head.

**Comment by sclevine** [comment]
We may want to move this to the lifecycle README, given https://github.com/buildpack/lifecycle/pull/14 and #27 .

**Comment by jkutner** [comment]
Indeed

**Pull request closed without merging**